### ISSUE TITLE
Offer website users option to use dyslexic friendly fonts E1304

### ISSUE THREAD

**Issue by rija** [gigascience/gigadb-website#1319]
## User story

>As a website user
>I want to be able to access all contents despite being dyslexic.
>So that I can comfortably access gigadb.org content

## Acceptance criteria

Given a user visits gigadb.org
When they choose the option to view page in dyslexic font
Then text on page is displayed in Opendyslexic font

## Additional Info

* https://opendyslexic.org

This is part of the Epic #1304 

GigaByte journal offer this service, its just a click button on each article (see [screenshot](https://gigabytejournal.com/articles/145)) that changes the font display to use a dyslexic freindly font.
![image](https://github.com/user-attachments/assets/76b6b27b-4845-478a-b66c-b1437c1df026)

## Product Backlog Item Ready Checklist

* [ ] Business value is clearly articulated
* [ ] Item is understood enough by the IT team so it can make an informed decision as to whether it can complete this item
* [ ] Dependencies are identified and no external dependencies would block this item from being completed
* [ ] At the time of the scheduled sprint, the IT team has the appropriate composition to complete this item
* [ ] This item is estimated and small enough to comfortably be completed in one sprint
* [ ] Acceptance criteria are clear and testable
* [ ] Performance criteria, if any, are defined and testable
* [ ] The Scrum team understands how to demonstrate this item at the sprint review

## Product Backlog Item Done Checklist

* [ ] Item(s) in increment pass all Acceptance Criteria
* [ ] Code is refactored to best practices and coding standards
* [ ] Documentation is updated as needed
* [ ] Data security has not been compromised (with particular reference to the personal information we hold in GigaDB)
* [ ] No deviation from the team technology stack and software architecture has been introduced
* [ ] The product is in a releasable state (i.e. the increment has not broken anything)

**Comment by luistoptal** [comment]
@only1chunts should this option be found in all pages including the admin portal or only is specific pages?

**Comment by only1chunts** [comment]
I dont know how complex the issue is, but ideally I think the option should be site-wide (all pages including admin), I know I certainly find it easier to read.
